activation/activation.go

```diff
@@ -320,13 +329,13 @@ func (b *Builder) MiningStats() (int, uint64, string, string) {
 // the rewards for blocks made by this miner will go to this address
 func (b *Builder) SetCoinbaseAccount(rewardAddress types.Address) {
 	b.accountLock.Lock()
-	b.coinbaseAccount = rewardAddress
+	b.conf.CoinbaseAccount = rewardAddress
```

Comment:
why not have a variable inside Builder for this? it looks a bit strange to change config from whithin, unless there is some logic I'm missing here

Reply:
I agree. It would be better to unwrap config to variables inside `Builder`.